web3deployment/basicDeployer.js

```diff
@@ -18,9 +18,7 @@ const web3 = new Web3(new Web3.providers.HttpProvider(rpcUrl));
 const solc = require('solc')
 
 const rand = web3.utils.randomHex(7);
-
 const privateKey = web3.utils.sha3("in joy we trust" + rand);
```

Comment:
Lol what.... Anton wrote this deployer?

Reply:
HAHA
no.
Actually it was originally "js sucks"
I wanted something more optimistic and it ended up like this...